File: tests/testdata/codegen/empty-list-property-pp/python/empty-list-property.py
import pulumi
import pulumi_azure_native as azure_native

storage_accounts = azure_native.storage.StorageAccount("storageAccounts",
    account_name="sto4445",
    kind=azure_native.storage.Kind.BLOCK_BLOB_STORAGE,
    location="eastus",
    resource_group_name="res9101",
    sku={
        "name": azure_native.storage.SkuName.PREMIUM_LRS,
    },
    network_rule_set={
        "default_action": azure_native.storage.DefaultAction.ALLOW,
        "ip_rules": [],
    })

File: tests/testdata/codegen/empty-list-property-pp/dotnet/empty-list-property.cs
using System.Collections.Generic;
using System.Linq;
using Pulumi;
using AzureNative = Pulumi.AzureNative;

return await Deployment.RunAsync(() => 
{
    var storageAccounts = new AzureNative.Storage.StorageAccount("storageAccounts", new()
    {
        AccountName = "sto4445",
        Kind = AzureNative.Storage.Kind.BlockBlobStorage,
        Location = "eastus",
        ResourceGroupName = "res9101",
        Sku = new AzureNative.Storage.Inputs.SkuArgs
        {
            Name = AzureNative.Storage.SkuName.Premium_LRS,
        },
        NetworkRuleSet = new AzureNative.Storage.Inputs.NetworkRuleSetArgs
        {
            DefaultAction = AzureNative.Storage.DefaultAction.Allow,
            IpRules = new() { },
        },
    });

});

